pkg/ui/workspaces/cluster-ui/src/indexDetailsPage/indexDetailsPage.tsx

```diff
@@ -384,15 +383,10 @@ export class IndexDetailsPage extends React.Component<
     return (
       <Breadcrumbs
```

Comment:
Im assuming that the database tables breadcrumb was removed because there is no database Id to use for the link. Should we fetch it via `/api/v2/table_metadata/<details.table_id>`? 

I think this would be useful to make the breadcrumb consistent with the other pages (will also allow for us to use the same table name as other pages, instead of the of the quoted table names like in the legacy pages

Reply:
It's a bit awkward to introduce this at the moment due to this being a legacy page. We could simplify it if we don't use SWR or just use the fetch call directly but I'm not sure if it's worth it. Alternative is to add the dbId to the index stats response. Thoughts? We can do that in a follow-up.